src/windows/video_widget.py

```diff
@@ -81,7 +81,7 @@ def paintEvent(self, event, *args):
         painter.setRenderHints(QPainter.Antialiasing | QPainter.SmoothPixmapTransform | QPainter.TextAntialiasing, True)
 
         # Fill the whole widget with the solid color
-        painter.fillRect(event.rect(), QColor("#191919"))
+        painter.fillRect(event.rect(), self.palette().window())
```

Comment:
I'm guessing the change was intentional — wanting to show the widget area better in the context of the zooming feature, probably. But even in that case, there's still a palette color that should be used:
```suggestion
        painter.fillRect(event.rect(), self.palette().base())
```
That'll use the same background color as the View widgets: `rgb(53, 53, 53)` for the dark theme, and `rgb(255, 255, 255)` for the light theme.

Reply:
> and `rgb(255, 255, 255)` for the light theme...

LOL @ferdnyc  this an eyesore color. Are you really suggesting this change?

Reply:
Window is window. There is nothing to reinvent.